app/models/setting/auth.rb

```diff
@@ -59,3 +59,4 @@ def validate_websockets_ssl_cert(record)
   end
 end
 
+require_dependency 'setting'
```

Comment:
why is this necessary? shouldn't this be implied by the class inheriting from Setting?